Sources/TuistLoader/Models+ManifestMappers/TargetDependency+ManifestMapper.swift

```diff
@@ -60,7 +61,7 @@ extension TuistGraph.TargetDependency {
         case .xctest:
             return [.xctest]
         case let .external(name):
-            guard let dependencies = externalDependencies[name] else {
+            guard let dependencies = externalDependencies[platform]?[name] else {
                 throw TargetDependencyMapperError.invalidExternalDependency(name: name)
```

Comment:
we should pass also the platform here to provide a more complete error message

Reply:
Enhanced the error message with the platform